containers/daap-delete-data-product/src/var/task/delete_data_product.py

```diff
@@ -0,0 +1,61 @@
+import os
+from http import HTTPStatus
+
+import boto3
+from data_platform_api_responses import format_error_response, format_response_json
+from data_platform_logging import DataPlatformLogger
+from data_product_metadata import DataProductMetadata
+from versioning import VersionCreator
+
+logger = DataPlatformLogger(
+    extra={
+        "image_version": os.getenv("VERSION", "unknown"),
+        "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+    }
+)
+
```

Comment:
```suggestion
logger = DataPlatformLogger()

```

As of 6.2.0 of the base, these versions are added automatically